app/views/council_memberships/index.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>My Councils</h1>
+<% @council_memberships.each do |membership| %>
+  <p><%= membership.council.council_name%></p>
```

Comment:
If you get time look up delegate. It lets you avoid method chains like this
http://apidock.com/rails/Module/delegate